<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255);
font: 15pt &quot;Times New Roman&quot;;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>605</width>
         <height>450</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_retrieve">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 248, 188);</string>
        </property>
        <property name="text">
         <string>Retrieve</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>59</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="Input_text">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>571</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255);
font: 14pt &quot;Times New Roman&quot;;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>59</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Input </string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_open">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 248, 188);</string>
        </property>
        <property name="text">
         <string>Insert File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_refresh">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 248, 188);</string>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="Output_text">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>571</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255);
font: 13pt &quot;Times New Roman&quot;;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_logo">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>-10</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Version 1.1.1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Output"/>
    <addaction name="actionSave_Input"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuContact">
    <property name="title">
     <string>Contact</string>
    </property>
    <addaction name="actionContact_Us"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuContact"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_Output">
   <property name="text">
    <string>Save Output</string>
   </property>
  </action>
  <action name="actionSave_Input">
   <property name="text">
    <string>Save Input</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionContact_Us">
   <property name="text">
    <string>Contact Us</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
